rest-server/conf/logback.xml

```diff
@@ -22,21 +23,20 @@
             <pattern>%date - [%level] - from %logger in %thread %n%message%n%xException%n</pattern>
         </encoder>
     </appender>
+    -->
 
-    <!--
-<appender name="FILE" class="ch.qos.logback.core.rolling.RollingFileAppender">
-    <file>${HARNESS_LOG_PATH}/harness.log</file>
-    <rollingPolicy class="ch.qos.logback.core.rolling.SizeAndTimeBasedRollingPolicy">
-        <fileNamePattern>harness-%d{yyyy-MM-dd}.%i.log</fileNamePattern>
-        <maxFileSize>1GB</maxFileSize>
-        <maxHistory>60</maxHistory>
-        <totalSizeCap>10GB</totalSizeCap>
-    </rollingPolicy>
-    <encoder>
-        <pattern>%d{HH:mm:ss.SSS} %-5level %-17logger{0} - %msg%n</pattern>
-    </encoder>
-</appender>
--->
+    <appender name="FILE" class="ch.qos.logback.core.rolling.RollingFileAppender">
+        <file>${HARNESS_LOG_PATH}/harness.log</file>
+        <rollingPolicy class="ch.qos.logback.core.rolling.SizeAndTimeBasedRollingPolicy">
+            <fileNamePattern>harness-%d{yyyy-MM-dd}.%i.log</fileNamePattern>
```

Comment:
yes, that's the place that I was talking about. This path is not the same as `${HARNESS_LOG_PATH}/harness.log`

Reply:
@andreyshikov this is the old config, yours is commented out.

If there is an error in this line what should it be?

Reply:
`${HARNESS_LOG_PATH}/harness-%d{yyyy-MM-dd}.%i.log` if you want them in the same dir

Reply:
aha, ok